tests/fuzzing/main.sh

```diff
@@ -28,8 +28,10 @@
 shopt -o -s nounset
 
 source fuzz-fpgaconf.sh
+source fuzz_fpgainfo.sh
```

Comment:
Please use fuzz-fpgainfo.sh as the file name for consistency.

Reply:
rename fuzz_fpgainfo.sh to fuzz-fpgainfo.sh